units/user-config-ovfenv.path

```diff
@@ -0,0 +1,10 @@
+[Unit]
```

Comment:
Let's go ahead and drop this. I realize that this is copied from one of the other paths, but it is very unlikely that we will hit this case (CoreOS is not designed to be run _in_ a container).